.goreleaser-template.yaml

```diff
@@ -595,7 +595,7 @@ checksum:
 #LINUXONLY#      - ghcr.io/{{ .Env.GITHUB_REPOSITORY_OWNER }}/openbao:{{ .Version }}-riscv64
 #LINUXONLY#      - ghcr.io/{{ .Env.GITHUB_REPOSITORY_OWNER }}/openbao:{{ .Version }}-s390x
 #LINUXONLY#  - name_template: ghcr.io/{{ .Env.GITHUB_REPOSITORY_OWNER }}/openbao:{{ .Major }}
-#LINUXONLY#    skip_push: false
+#LINUXONLY#    skip_push: ${{ .Prerelease }}
```

Comment:
Notably, this will make it hard to do point releases for security issues in older branches when we have a newer GA major major version, given we have no way of pushing tags manually.

However, the TSC approved a strictly linear support policy so I think we're good to do so.